### PR TITLE
Homework2: Natallia Rogova

### PR DESCRIPTION
**Pull request by nrogova**

### CODE REVIEW

File: src/homework2_task6_extra.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'''
+Определите, является ли число палиндромом (читается слева направо и справа налево одинаково).  Число положительное целое, произвольной длинны.
+'''
+#variables
+num = int(input('Enter number '))
+#logic
+if str(num) == str(num)[::-1]:
```
Comment (belovmd) [inline src/homework2_task6_extra.py]: Please try to implement it without strings.

**Review by belovmd — CHANGES_REQUESTED** [review]
Please fix codestyle and take a look at issue below

File: src/homework2_task6_extra.py (hunk 1 above)
Comment (nrogova) [reply src/homework2_task6_extra.py]: Done.

**Review by belovmd — COMMENTED** [review]
8

**Pull request closed without merging**